ldi-core/sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/utils/SparqlFunctionsUtils.java

```diff
@@ -0,0 +1,102 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.utils;
+
+import org.apache.jena.geosparql.implementation.GeometryWrapper;
+import org.apache.jena.geosparql.implementation.GeometryWrapperFactory;
+import org.apache.jena.graph.Node;
+import org.apache.jena.sparql.expr.NodeValue;
+import org.locationtech.jts.geom.Coordinate;
+
+import java.util.stream.DoubleStream;
+import java.util.stream.IntStream;
+
+public class SparqlFunctionsUtils {
+
+	static double SEMI_MAJOR_AXIS_MT = 6378137;
+	static double SEMI_MINOR_AXIS_MT = 6356752.314245;
+	static double FLATTENING = 1 / 298.257223563;
+	static double ERROR_TOLERANCE = 1e-12;
+
+	/**
+	 * Vincenty's Formula
+	 */
+	public static double calculateDistance(double latitude1, double longitude1, double latitude2, double longitude2) {
```

Comment:
I'm just going to believe you on this one ;) However can you maybe add to the javadoc what exactly we get back? Is the distance in meters, km, miles, etc. ?